addons/im_livechat/tests/test_digest.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.digest.tests.common import TestDigestCommon
+from odoo.tools import mute_logger
+from odoo.tests import tagged
+
+
+@tagged('post_install')
+class TestLiveChatDigest(TestDigestCommon):
+
+    @classmethod
+    @mute_logger('odoo.models.unlink')
+    def setUpClass(cls):
+        super(TestLiveChatDigest, cls).setUpClass()
+
+        other_partner = cls.env['res.partner'].create({'name': 'Other Partner'})
+
+        cls.channels = cls.env['mail.channel'].create([{
+            'name': 'Channel 1',
+            'livechat_operator_id': cls.env.user.partner_id.id,
+        }, {
+            'name': 'Channel 2',
+            'livechat_operator_id': cls.env.user.partner_id.id,
+        }, {
+            'name': 'Channel 3',
+            'livechat_operator_id': other_partner.id,
+        }])
+
+        cls.env['rating.rating'].create([{
+            'rated_partner_id': cls.env.user.partner_id.id,
+            'res_id': cls.channels[0].id,
+            'res_model_id': cls.env['ir.model']._get('mail.channel').id,
+            'consumed': True,
+            'rating': 5,
+        }, {
+            'rated_partner_id': cls.env.user.partner_id.id,
+            'res_id': cls.channels[0].id,
+            'res_model_id': cls.env['ir.model']._get('mail.channel').id,
+            'consumed': True,
+            'rating': 0,
+        }, {
+            'rated_partner_id': cls.env.user.partner_id.id,
+            'res_id': cls.channels[0].id,
+            'res_model_id': cls.env['ir.model']._get('mail.channel').id,
+            'consumed': True,
+            'rating': 3,
+        }, {
+            'rated_partner_id': other_partner.id,
+            'res_id': cls.channels[0].id,
+            'res_model_id': cls.env['ir.model']._get('mail.channel').id,
+            'consumed': True,
+            'rating': 5,
+        }])
+
+    def test_kpi_livechat_rating_value(self):
+        self.assertEqual(round(self.digest_1.kpi_livechat_rating_value, 2), 33.33)
```

Comment:
Is that 33%? Can you make a quick explanation of how you get that with the initial data set? It's not super obvious ^^